web.Dockerfile

```diff
@@ -1,24 +1,21 @@
 # You can set the Swift version to what you need for your app. Versions can be found here: https://hub.docker.com/_/swift
-FROM swift:4.2 as builder
+FROM swift:5.0 as builder
 
 # For local build, add `--build-arg env=docker`
 # In your application, you can use `Environment.custom(name: "docker")` to check if you're in this env
 ARG env
 
-RUN apt-get -qq update && apt-get -q -y install \
-  tzdata \
-  && rm -r /var/lib/apt/lists/*
```

Comment:
I think `rm -r /var/lib/apt/lists/*` is there to reduce the overall image size. Maybe we should keep it?